explorer/explorerroutes.go

```diff
@@ -1837,14 +1837,30 @@ func (exp *explorerUI) ProposalPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var timeLeft string
+	endHeight, _ := strconv.ParseInt(proposalInfo.ProposalVotes.Endheight, 10, 64)
```

Comment:
Can this never error?  Please add a comment describing why that is the case if it is so.